mrpt_map/src/map_server_node.cpp

```diff
@@ -88,14 +88,14 @@ void MapServer::init()
 	metric_map_ = boost::shared_ptr<CMultiMetricMap>(new CMultiMetricMap);
 	mrpt_bridge::MapHdl::loadMap(
 		*metric_map_, config_file, map_file, "metricMap", debug_);
-	const COccupancyGridMap2D& grid = *metric_map_->m_gridMaps[0];
+	const COccupancyGridMap2D& grid = *metric_map_->mapByClass<COccupancyGridMap2D>();
 	if (debug_)
 		printf(
 			"gridMap[0]:  %i x %i @ %4.3fm/p, %4.3f, %4.3f, %4.3f, %4.3f\n",
 			grid.getSizeX(), grid.getSizeY(), grid.getResolution(),
 			grid.getXMin(), grid.getYMin(), grid.getXMax(), grid.getYMax());
 
-	mrpt_bridge::convert(*metric_map_->m_gridMaps[0], resp_ros_.map);
+	mrpt_bridge::convert(*metric_map_->mapByClass<COccupancyGridMap2D>(), resp_ros_.map);
 	if (debug_)
```

Comment:
Thanks!

Should we also have here a

	#if MRPT_VERSION >= 0x199

?

Reply:
Yes I think so, will update it